packages/dds/merge-tree/src/index.ts

```diff
@@ -133,7 +112,7 @@ export { SegmentGroupCollection } from "./segmentGroupCollection";
 export { PropertiesManager, PropertiesRollback } from "./segmentPropertiesManager";
 export { SortedSet } from "./sortedSet";
 export { SortedSegmentSet, SortedSegmentSetItem } from "./sortedSegmentSet";
-export { IJSONTextSegment, IMergeTreeTextHelper, TextSegment } from "./textSegment";
```

Comment:
the typing is not used, but we kind do export this type, see TextSegment.toJSONObject, i think i'd rather see that method return type explicitly typed, as we depend on round trip